emailproxy.py

```diff
@@ -91,6 +91,16 @@ class NSObject:
             pass
 del no_gui_parser
 
+# by default the proxy stores the OAuth 2.0 tokens in a local config file, but it can optionally store them
+# remotely in AWS Secrets Manager. import the dependency only if this option is specified.
+aws_secrets_parser = argparse.ArgumentParser()
```

Comment:
Exclude this ArgumentParser from `--help`

```suggestion
aws_secrets_parser = argparse.ArgumentParser(add_help=False)
```